toonz/sources/common/tsystem/tsystempd.cpp

```diff
@@ -385,6 +436,31 @@ TINT64 TSystem::getMemorySize(bool onlyPhisicalMemory) {
   free(sysInfo);
   return ret;
 
+#elif defined(FREEBSD)
+
+  TINT64 ret = 0;
+#ifdef __OpenBSD__
+  int mib[] = {CTL_VM, VM_UVMEXP};
+  struct uvmexp  uvmexp;
+#else
+  int mib[] = {CTL_VM, VM_TOTAL};
+  struct vmtotal vmtotal;
+#endif
+  size_t size;
+#ifdef __OpenBSD__
```

Comment:
Why are there `OpenBSD` definitions here? Should they perhaps follow the style of the other defines? `#if defined(OPENBSD)`

If you've seen code like this in other places, might be good to leave a link referring to them.

Reply:
Cmake check ```elseif(CMAKE_SYSTEM_NAME MATCHES "^.*BSD$|DragonFly")``` and defines ```FREEBSD``` for any BSD based system, except MAC OS.
I hope after this PR OpenToonz will work on any BSD system, probably with minimal changes for non FreeBSD.
FreeBSD defines ```__FreeBSD__```, OpenBSD ```__OpenBSD__``` and vice versa some where in standard includes.

Reply:
Ok!

Just for pedantry's sake, it's usually the compiler that's responsible for those defines ;)